src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -541,6 +541,43 @@ public void canSearchUsingKeywordIndexAll()
     assertThat(allInstances.size(), is(1));
   }
 
+  @Test
+  public void canSearchUsingMetadataDateUpdatedIndexAll()
+    throws MalformedURLException,
+    InterruptedException,
+    ExecutionException,
+    TimeoutException {
+
+    UUID firstInstanceId = UUID.randomUUID();
+
+    JsonObject firstInstanceToCreate = smallAngryPlanet(firstInstanceId);
+
+    createInstance(firstInstanceToCreate);
+
+    UUID secondInstanceId = UUID.randomUUID();
+
+    JsonObject secondInstanceToCreate = nod(secondInstanceId);
+
+    IndividualResource ir = createInstance(secondInstanceToCreate);
+
+    CompletableFuture<Response> getCompleted = new CompletableFuture<>();
+
+    JsonObject metadata = ir.getJson().getJsonObject("metadata");
+
+    String query = urlEncode(String.format("%s.updatedDate>=%s", METADATA_KEY, metadata.getString("updatedDate")));
+
+    client.get(instancesStorageUrl(String.format("?query=%s", query)), StorageTestSuite.TENANT_ID,
+        ResponseHandler.json(getCompleted));
+
+    Response response = getCompleted.get(5, TimeUnit.SECONDS);
+
+    JsonObject responseBody = response.getJson();
+
+    JsonArray allInstances = responseBody.getJsonArray("instances");
+
+    assertThat(allInstances.size(), is(1));
```

Comment:
Could two instances be created and filter so that only one of them is returned in the results?

With the way that the CQL searching works, if an index name is incorrect, it is effectively ignored, which I think means this test could easily produce a false positive (inclusion of the instance because it is the only one in the system)

Reply:
I believe this test is creating two instances and then only returning the second from the search. Is that not the case?

Reply:
My apologies, indeed it is. Please could an assertion be added to check that the expected instance is included in the results